src/test/java/com/wikia/webdriver/elements/communities/desktop/pages/CreateNewWikiPage.java

```diff
@@ -0,0 +1,208 @@
+  @FindBy(css = ".wiki-base-domain")
+  private WebElement domainSufix;
+  @FindBy(css = ".wiki-domain-error.error-msg")
+  private WebElement wikiDomainErrorMessage;
+  @FindBy(css = "#DescWiki .wds-dropdown")
+  private WebElement wikiCategoryDropdown;
+  @FindBy(css = "#DescWiki .wds-dropdown .wds-list li")
+  private List<WebElement> wikiCategoryList;
+  @FindBy(css = "form[name='desc-form'] input.next")
+  private WebElement createMyWikiBautton;
+  @FindBy(css = "label[for='allAges']")
+  private WebElement allAgesCheckBox;
+  @FindBy(css = "#DescWiki .wiki-vertical-error.error-msg")
+  private WebElement categoryErrorMsg;
+  @FindBy(css = "li[id='ThemeWiki'] input.next.enabled")
+  private WebElement submitButton;
+  @FindBy(css = "#CreateNewWiki")
+  private WebElement wikiTaskId;
+
+  /**
+   * Open special Page to create new Wikia. This special page 'Special:CreateNewWiki' is only
+   * available on community.fandom.com domain
+   */
+  public CreateNewWikiPage open() {
+    getUrl(
+        UrlBuilder.createUrlBuilderForWiki("community")
+            .getUrlForWikiPage(SPECIAL_CREATE_NEW_WIKI_PAGE));
+
+    return this;
+  }
+
+  public String getWikiName() {
+    return CreateWikiMessages.WIKINAME_PREFIX + DateTime.now().getMillis();
+  }
+
+  public String getDomainSufix() {
+    return domainSufix.getText();
+  }
+
+  public CreateNewWikiPage selectLanguage(String lang) {
+    jsActions.scrollToElement(wait.forElementClickable(wikiLanguageDropdown));
+    String langSelector = lang + ":";
+
+    WebElement langElementInDropdown = wikiLanguageList.stream()
+        .filter(e -> e.getAttribute("innerHTML").trim().contains(langSelector))
+        .findAny()
+        .orElseThrow(() -> new WebDriverException(String.format("Couldn't find language [%s]",
+                                                                lang
+        )));
+    hover(wikiLanguageDropdown);
+    langElementInDropdown.click();
+
+    Log.log("selectLanguage",
+            "selected " + langElementInDropdown.getAttribute("innerHTML").trim() + " language",
+            true,
+            driver
+    );
+
+    return this;
+  }
+
+  public CreateNewWikiPage typeInWikiName(String name) {
+    wikiName.sendKeys(name);
+    Log.log("typeInWikiName ", "Typed wiki name" + name, true);
+
+    return this;
+  }
+
+  public CreateNewWikiPage typeInWikiDomain(String domain) {
+    wikiDomain.clear();
+    wikiDomain.sendKeys(domain);
+    Log.log("typeInWikiDomain ", "Typed wiki domain " + domain, true);
+
+    return this;
+  }
+
+  public String getDomainErrorMessage() {
+    wait.forTextNotEmpty(wikiDomainErrorMessage);
+
+    return wikiDomainErrorMessage.getText().toLowerCase();
+  }
+
+  public CreateNewWikiPage clickNext() {
+    wait.forElementClickable(nextButton);
+    Log.log("waitForNextButton", "Next button enabled", true, driver);
+
+    scrollAndClick(nextButton);
+    Log.log("submit", "Submit button clicked", true, driver);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "submit" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Submit button clicked" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)